src/components/admin/settings-tab.js

```diff
@@ -0,0 +1,170 @@
+      onDirtyStateChange(false);
+    } catch (error) {
+      handleApiError({ error, logger, t });
+    }
+  };
+
+  const handleCreateDocumentRegenerationBatchClick = async () => {
+    try {
+      await documentApiClient.postDocumentRegenerationBatch();
+    } catch (error) {
+      handleApiError({ t, logger, error });
+    }
+  };
+
+  const templateDocumentURL = settings.templateDocument
+    ? `${settings.templateDocument.documentKey}/${settings.templateDocument.documentSlug}`
+    : '';
+
+  return (
+    <div className="SettingsTab">
+      <Card className="Card" title={t('homepageInfoHeader')}>
+        <MarkdownTextarea
+          value={settings.homepageInfo || ''}
+          onChange={handleHomepageInfoChange}
+          />
+      </Card>
+      <Card className="Card" title={t('templateDocumentHeader')}>
+        <DocumentSelector
+          documents={documents}
+          value={templateDocumentURL}
+          onChange={handleTemplateDocumentChange}
+          />
+      </Card>
+      <Card className="Card" title={t('helpPageHeader')}>
+        <SpecialPageSettings
+          settings={settings.helpPage}
+          documents={documents}
+          onChange={handleHelpPageChange}
+          />
+      </Card>
+      <Card className="Card" title={t('termsPageHeader')}>
+        <SpecialPageSettings
+          settings={settings.termsPage}
+          documents={documents}
+          onChange={handleTermsPageChange}
+          />
+      </Card>
+      <Card className="Card" title={t('footerLinksHeader')}>
+        <FooterLinksSettings
+          footerLinks={settings.footerLinks}
+          documents={documents}
+          onChange={handleFooterLinksChange}
+          />
+      </Card>
+      <Card className="Card" title={t('defaultTagsHeader')}>
+        <DefaultTagsSettings
+          defaultTags={settings.defaultTags || []}
+          onChange={handleDefaultTagsChange}
+          />
+      </Card>
+      <Card className="Card" title={t('licenseHeader')}>
+        <LicenseSettings
+          license={settings.license}
+          onChange={handleLicenseChange}
+          />
+      </Card>
+      <Card className="Card Card--danger" title={t('createDocumentRegenerationBatchHeader')}>
+        <Button
+          onClick={handleCreateDocumentRegenerationBatchClick}
+          danger
+          >
+          {t('createDocumentRegenerationBatchButton')}
+        </Button>
+      </Card>
+      <Button
+        type="primary"
+        onClick={handleSaveButtonClick}
+        className="SettingsTab-saveButton"
+        disabled={!!invalidKeys.length || !dirtyKeys.length}
+        >
+        {t('common:save')}
+      </Button>
+    </div>
+  );
+}
+
+SettingsTab.propTypes = {
+  documents: PropTypes.arrayOf(documentMetadataShape).isRequired,
+  initialSettings: settingsShape.isRequired,
+  onDirtyStateChange: PropTypes.func.isRequired,
+  onSettingsSaved: PropTypes.func.isRequired
+};
+
+export default SettingsTab;
```

Comment:
This tab looks so much more readable now with cards! 👍

Reply:
Completely agree :grin: